glide.yaml

```diff
@@ -50,6 +50,6 @@ import:
 - package: github.com/emicklei/go-restful
   version: v1.2
 - package: github.com/containernetworking/cni
-  version: ^0.5.2
+  version: v0.5.2
```

Comment:
Since v0.6.0-rc1 was released a couple weeks ago, any time that glide updates, it grabs the newest version (and does not skip rc) so this was grabbing a version that no longer has `pkg/ns` package that some of the logic depends on.  We eventually should update to reflect those changes, but for this change, it is unnecessary, so we should pin the version until we are ready to handle the newer one.